### ISSUE TITLE
Error when detecting face

### ISSUE THREAD

**Issue by ScottDellinger** [justadudewhohacks/face-api.js#647]
Occassionally, we receive an error when attempting to detect a face in a still image, we receive the following error:

Uncaught (in promise) Error: Box.constructor - expected box to be IBoundingBox | IRect, instead have {"left":211.82223243423,"top":188.34445656565,"right":299.35465223232,"bottom":null}
at BoundingBox.Box
at new BoundingBox
at minBbox
at FaceLandmarks68.FaceLandmarks.alignMinBbox
...

It appears that it's missing a value for the bottom of a bounding box, but I'm not sure why. Any suggestions?

**Comment by felipebutcher** [comment]
I was having this error when running the html served in a node environment. Running it from traditional apache works fine.

**Comment by ha6755ad** [comment]
Hi @ScottDellinger Did you ever manage to find a solution for this issue?

**Comment by ScottDellinger** [comment]
> Hi @ScottDellinger Did you ever manage to find a solution for this issue?

No... I stopped using this library because it's been abandoned and we found the models it was trained with did not perform well on Asian or Middle Eastern faces. There is a more active fork of it, though.

**Comment by Ghos1ware** [comment]
> 

Can you share the link ? Can't seem to find anything active.

**Comment by Ghos1ware** [comment]
> > Hi @ScottDellinger Did you ever manage to find a solution for this issue?
> 
> No... I stopped using this library because it's been abandoned and we found the models it was trained with did not perform well on Asian or Middle Eastern faces. There is a more active fork of it, though.


Can you share the link ? Can't seem to find anything active.

**Comment by ScottDellinger** [comment]
> > > Hi @ScottDellinger Did you ever manage to find a solution for this issue?
> > 
> > 
> > No... I stopped using this library because it's been abandoned and we found the models it was trained with did not perform well on Asian or Middle Eastern faces. There is a more active fork of it, though.
> 
> Can you share the link ? Can't seem to find anything active.

https://github.com/vladmandic/face-api

**Comment by Ghos1ware** [comment]
> > > > Hi @ScottDellinger Did you ever manage to find a solution for this issue?
> > > 
> > > 
> > > No... I stopped using this library because it's been abandoned and we found the models it was trained with did not perform well on Asian or Middle Eastern faces. There is a more active fork of it, though.
> > 
> > 
> > Can you share the link ? Can't seem to find anything active.
> 
> https://github.com/vladmandic/face-api

Thank you Scott! ;)

**Comment by rubaiyet-anjum** [comment]
> > > > Hi @ScottDellinger Did you ever manage to find a solution for this issue?
> > > 
> > > 
> > > No... I stopped using this library because it's been abandoned and we found the models it was trained with did not perform well on Asian or Middle Eastern faces. There is a more active fork of it, though.
> > 
> > 
> > Can you share the link ? Can't seem to find anything active.
> 
> https://github.com/vladmandic/face-api

Thanks as well ☺️
